src/main/java/frc/robot/subsystems/drive/SwerveModules.java

```diff
@@ -87,7 +88,7 @@ public double getDriveVelocityMPerS() {
     }       
 
     public double getDriveVelocityRadPerS() {
-        return driveMotor.getEncoder().getVelocity() * 2 * Math.PI; 
+        return driveMotor.getEncoder().getVelocity() * 2 * Math.PI / 60; 
```

Comment:
@redPlover we actually need to divide by driveWheelGearReduction here as well because the encoder velocity is going to be a different speed than the wheel itself due to the gear reduction

Reply:
Done